wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/ConversationGuestOptionsViewController.swift

```diff
@@ -20,6 +20,10 @@ import UIKit
 import WireDataModel
 import WireSyncEngine
 
+extension NSNotification.Name {
+    static let didCreateSecureGuestLink = NSNotification.Name("didCreateSecureGuestLink")
+}
```

Comment:
Do we need to extend NSNotification here? Following `UIApplication.willEnterForegroundNotification` I would suggest to add a feature namespace or similar like:
```swift
enum Conversation {
    static let didCreateSecureGuestLinkNotification = Notification.Name("Conversation.didCreateSecureGuestLink")
}
```

Reply:
I don't really follow, can you share more about it?

Reply:
Nvm, I fixed it. https://github.com/wireapp/wire-ios/pull/1339/commits/a5491d4b9df90b2c3e32b3b84857057006e4da59